networkdb/networkdb.go

```diff
@@ -1,5 +1,7 @@
 package networkdb
 
+//go:generate protoc -I.:../Godeps/_workspace/src/github.com/gogo/protobuf  --gogo_out=import_path=github.com/docker/libnetwork/networkdb,Mgogoproto/gogo.proto=github.com/gogo/protobuf/gogoproto:. networkdb.proto
+
```

Comment:
@mrjana I am not that familiar with gogo protobuf. What does this `Mgogoproto/gogo.proto=github.com/gogo/protobuf/gogoproto` do ?